### ISSUE TITLE
Clicking association link does not work across servers

### ISSUE THREAD

**Issue by roverwolf** [opensalt/opensalt#324]
When as association is referencing a different server, the URL generated to get to it is not valid as the javascript currently takes the URI of the remote document (.../ims/case/v1p0/CFDocuments/*doc identifier*) and treats all but the *doc identifier* as a "base url" which the identifier of the associated item is appended to (to get .../ims/case/v1p0/CFDocuments/*item identifier*).

The generated link needs to either take the URI of the item and use it directly (which then points to the CASE JSON API page), or modify it to go to the **/uri/*item identifier*** link which will redirect to the proper page to view the item (which assumes the remote server is OpenSALT).

Unfortunately the CASE API does not require the use of an accept header indicating *application/json* or we could redirect *text/html* requests to the appropriate rendered item page.

We could possibly look at adding a non-standard query parameter on the API call that would indicate to OpenSALT that it should redirect the request, or we could save some information within the instance so that it would know that the remote server is an OpenSALT instance and can reformat the destination URL in a known way.

**Comment by BarbAllen** [comment]
Is this related to #295?

**Comment by roverwolf** [comment]
No.  #295 has to do with getting the framework data into the UI.  This is to do with the outgoing link that comes from that data.

**Comment by roverwolf** [comment]
A short-term fix was added in [commit 07020d5](https://github.com/opensalt/opensalt/commit/07020d5b38d7ea4180b33ed2893b0d620a9142f1#diff-9d609b6222905cf17f740eda4a43364cR124) (in 2.0), but it assumes that the other server is an OpenSALT server.
